roles/st2chatops/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: Install latest st2chatops package
+  become: yes
+  package:
+    name: st2chatops
+    state: latest
+  when: st2chatops_version == "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Install pinned st2chatops package
+  become: yes
+  package:
+    name: st2chatops={{ st2chatops_version }}
+    state: present
+  when: st2chatops_version != "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Add st2_api_key in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '(?<=ST2_API_KEY=)(.*)$'
+    replace: '{{ st2chatops_config.ST2_API_KEY }}'
+  when: st2chatops_config.ST2_API_KEY != "CHANGE-ME-PLEASE"
+  register: st2chatops_api_key
+  tags: st2chatops
+
+- name: Comment Username, Password and Auth URL, if API_KEY provided
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '{{ item }}'
+    replace: '# \1'
+  with_items:
+    - '^(export ST2_AUTH_URL.*)'
+    - '^(export ST2_AUTH_USERNAME.*)'
+    - '^(export ST2_AUTH_PASSWORD.*)'
+  when: st2chatops_api_key.changed
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Uncomment Hubot Slack Adapter in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '^# (export HUBOT_ADAPTER=slack)$'
+    replace: '\1'
+  when: st2chatops_api_key.changed and st2chatops_config.HUBOT_ADAPTER == "slack"
+  register: uncomment_slack_adapter
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Uncomment and replace Hubot Slack Token in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '^# (export HUBOT_SLACK_TOKEN.).*'
```

Comment:
The problem with that approach is described here: https://github.com/StackStorm/ansible-st2/pull/125#discussion_r104658131

Ex: @emedvedev merges Mattermost support (https://github.com/StackStorm/st2chatops/pull/65/files) and we obviously won't include it in `ansible-st2`.

Another example is that the list of vars we list in `st2chatops.env` is not full and can't be. There are a bunch of custom env vars for this or that adapter and even Hubot itself.
Just a quick example from search:
- https://github.com/github/hubot/blob/498d65611388d2881e6ae24f74bd68a832a24211/src/robot.coffee#L413-L417
- https://github.com/github/hubot/blob/498d65611388d2881e6ae24f74bd68a832a24211/bin/hubot#L24-L30

Who knows about more vars hidden in the code?
I don't even want to care about it, it's user's problem. They know what they need, - let's give an instrument.

> BTW this approach is similar to https://github.com/StackStorm/ansible-st2/pull/121 to avoid relying on `st2.conf` templating which can become outdated quickly.